businessRules/donations.js

```diff
@@ -21,34 +23,26 @@ function createDonation(firstName, sum, userId, ideaId) {
 }
 
 function ideasDonationSum(ideaId) {
-    const ideas = getAllIdeas();
-    const idea = ideas.find((idea) => idea.id === ideaId);
+    let idea = ideaWithTotalDonationSumThatMatchesIdeaId(ideaId);
```

Comment:
See if you can adopt this naming pattern:
1. Any function that returns a value other than `undefined` or `null` should have a name that starts with `get` or `find` or `filter` or `sort`, etc. - it should start with a verb.
2. Any function that mutates a value should have a name that starts with `set`, `update`, `delete`, `create`, etc. - it should start with a verb as well.

In general - functions are actions - they do something. So when you're thinking about how to name a function - think about what it does and try to communicate that in the name itself.

Reply:
Renamed.